libbeat/scripts/Makefile

```diff
@@ -95,6 +96,10 @@ ifeq ($(RACE_DETECTOR),1)
 	RACE=-race
 endif
 
+# Only build experimental targets for snapshots
+ifeq ($(SNAPSHOT),no)
```

Comment:
All of the other checks against this value look for `yes`. So I think it would be better to do an "ifneq" to "yes" check.